.github/CONTRIBUTING.md

```diff
@@ -16,48 +18,51 @@ Welcome! We are really excited that you are interested in contributing to Condui
 
-- Open a new issue using the [Issue Template](https://github.com/ConduitPlatform/Conduit/blob/master/.github/ISSUE_TEMPLATE.md)
-
 ## Pull Request Guidelines
 
-- Checkout a topic branch from `master`, and merge back.
+- Checkout a topic branch from `main`, and merge back.
 
 - It's OK to have multiple small commits as you work on the PR - GitHub will automatically squash it before merging.
 
 - If adding a new feature:
     - Add accompanying test case.
     - Provide a convincing reason to add this feature. Ideally, you should open a suggestion issue first and have it approved before working on it.
 
-- If fixing bug:
+- If fixing a bug:
     - If you are resolving a reported issue, add `(fix #xxx)` (#xxx is the issue id) in your PR title for a better release log, e.g. `update entities encoding/decoding (fix #389)`.
     - Provide a detailed description of the bug in the PR and the steps to reproduce it.
 
 - **DO NOT** change `.gitignore`.
 
-- Open a new pull request using the [Pull Request Template](https://github.com/ConduitPlatform/Conduit/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
+## Development Setup 🔨
 
-## Development Setup
-
-You will need [Node.js](http://nodejs.org) **version 14+**, [yarn](https://yarnpkg.com/en/docs/install) and either [MongoDB](https://www.mongodb.com/) or [PostgreSQL](https://www.postgresql.org/).
+### Requirements:
+- Node.js >=16
+- [protoc](https://grpc.io/docs/protoc-installation)
+- MongoDB or PostgreSQL
+- Redis
+- Desire to create something awesome
 
 After cloning the repo, run:
 
 ``` bash
-yarn
+yarn --frozen-lockfile
 npx lerna run build
 REDIS_HOST=localhost REDIS_PORT=6379 yarn --cwd ./packages/core start
-CONDUIT_SERVER=0.0.0.0:55152 SERVICE_IP=0.0.0.0:55165 yarn --cwd ./modules/database start
+CONDUIT_SERVER=0.0.0.0:55152 SERVICE_URL=0.0.0.0:55165 DB_CONN_URI=mongodb://localhost:27017 yarn --cwd ./modules/database start
 ```
 
-Then repeat the last step for every additional module you need to bring online:
+Then repeat the following step for every additional module you wish to bring online, specifying any additional env vars.
 
 ``` bash
-CONDUIT_SERVER=0.0.0.0:55152 SERVICE_IP=0.0.0.0:PORT yarn --cwd ./modules/MODULE start
+CONDUIT_SERVER=0.0.0.0:55152 SERVICE_URL=0.0.0.0:PORT yarn --cwd ./modules/MODULE start
 ```
 
+You may look up supported envs and configuration options for your modules in the [modules section of the documentation](https://getconduit.dev/docs/modules).
+
 ### Committing Changes
 
-Commit messages should follow the [commit message convention](https://github.com/ConduitPlatform/Conduit/blob/master/.github/COMMIT_CONVENTION.md) so that changelogs can be automatically generated.
+Commit messages should follow the [commit message convention](https://github.com/ConduitPlatform/Conduit/blob/main/.github/COMMIT_CONVENTION.md) so that changelogs can be automatically generated.
 
 ### Commonly Used Commands
```

Comment:
scope is prefixed with '@conduitplatform'

Reply:
Nice catch 👍